tenable/io/v3/platform/groups/api.py

```diff
@@ -0,0 +1,250 @@
+'''
+Groups
+======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`groups <groups>` API.
+
+Methods available on ``tio.v3.platform.groups``:
+
+.. rst-class:: hide-signature
+.. autoclass:: GroupsAPI
+    :members:
+'''
+from typing import Dict, List, Union
+from uuid import UUID
+
+from requests import Response
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+
+
+class GroupsAPI(ExploreBaseEndpoint):
+    '''
+    This will contain all methods related to Groups
+    '''
+
+    _path = 'api/v3/groups'
+    _conv_json = True
+
+    def add_user(self, group_id: UUID, user_id: UUID) -> List:
+        '''
+        Add a user to a user group.
+
+        :devportal:`groups: add-user <groups-add-user>`
+
+        Args:
+            group_id (UUID):
+                The unique identifier of the group to add the user to.
+            user_id (UUID):
+                The unique identifier of the user to add.
+
+        Returns:
+            List:
+                List of user resource records based on membership to the
+                specified group.
+
+        Examples:
+            >>> tio.v3.platform.groups.add_user(
+            ...     '00000000-0000-0000-0000-000000000000',
+            ...     '00000000-0000-0000-0000-000000000000'
+            ... )
+        '''
+        self._post(f'{group_id}/users/{user_id}')
```

Comment:
Add validation against proper value being passed for **group_id** and **user_id**.

Reply:
Added validation